docs/src/app/api/internal/[transport]/route.ts

```
@@ -220,6 +236,42 @@ const handler = createMcpHandler(
         }
       }
     );
+    server.tool(
+      "get_stack_auth_setup_instructions",
+      "Use this tool when the user wants to set up authentication in a new project. It provides step-by-step instructions for installing and configuring Stack Auth authentication.",
+      {},
+      async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)